app/forms/classification_form2.py

```diff
@@ -0,0 +1,13 @@
+from flask_wtf import FlaskForm
+from wtforms import SubmitField, SelectField
+from wtforms.validators import DataRequired
+
+from app.utils.list_images import list_images
+from config import Configuration
+
+conf = Configuration()
+
+
+class ClassificationForm2(FlaskForm):
```

Comment:
This lacks some documentation. What is the difference from the original ClassificationForm? Why do we need to introduce this new element? Please add a docstring or name it in a way that makes its function clear.